src/db/sqlite/lrsql/sqlite/sql/delete.sql

```diff
@@ -59,3 +59,53 @@ DELETE FROM credential_to_scope
 WHERE api_key = :api-key
 AND secret_key = :secret-key
 AND scope = :scope
+
+-- :name delete-actor-st2actor!
+-- :command :execute
+-- :result :affected
+DELETE FROM statement_to_actor where actor_ifi = :actor-ifi
+
+-- :name delete-actor-st2st!
+-- :command :execute
+-- :result :affected
+DELETE FROM statement_to_statement WHERE descendant_id in (:del-statement-ids)
+
+-- :name delete-actor-st-parent-rels!
+-- :command :execute
+-- :result :affected   
+DELETE FROM statement_to_statement WHERE descendant_id in (:del-statement-ids)
+
+-- :name delete-actor-st-desc-rels!
+-- :command :execute
+-- :result :affected   
+DELETE FROM statement_to_statement WHERE ancestor_id in (:del-statement-ids)
+
+-- :name delete-actor-st2activ!
+-- :command :execute
+-- :result :affected   
+DELETE FROM statment_to_activity sta 
+LEFT JOIN xapi_statement ON sta.statement_id=xapi_statement.statement_id 
+WHERE statement_to_activity.statement_id IN (:del-statement-ids)
+
+-- :name delete-actor-delete-reactions!
+-- :command :execute
+-- :result :affected   
+DELETE FROM reactions 
+LEFT JOIN xapi_statement ON reactions.id = xapi_statement.reaction_id 
+WHERE xapi_statement.statement_id IN (:del-statement-ids)
```

Comment:
I'm not sure if we should implement cascades on reactions when they're not fully implemented yet. @milt what do you think?

Reply:
No definitely not. Also, i cannot devise a way in which reactions would need to be deleted at all, even when thats merged in. This is looking at it backwards. Reaction id is just a tag on a statement, you can safely delete the statement and the data goes away.

Reply:
What Cliff said basically. We use soft-delete on reactions because statements have the FK, but in the other direction it's fine, the statement can just be deleted.